README.md

```diff
@@ -1,11 +1,7 @@
 # Stoat
 
-A GitHub action to supercharge your CI / CD pipelines.
+Turn PR comments into developer dashboards with templating and artifact previews for code coverage reports, frontend components, doc sites, and more! [Visit our docs website to learn more ↗︎](https://docs.stoat.dev/)
 
-## Local development
+---
 
-```bash
-npm install
-npm run build && npm run package
-npm test
-```
```

Comment:
I think it's still alright to include dev notes in the README.

It's fine though. I can add it back as I add the script for releasing new versions.

Reply:
Yeah we'll want maintenance stuff, how to contribute, etc.